_api/hasura/cron/syncCoSouls.ts

```diff
@@ -171,7 +174,8 @@ const syncCoSoulToken = async (
     totalPGIVE = Math.floor(totalPGIVE);
     const txHash = await setOnChainPGive({ tokenId, amount: totalPGIVE });
 
-    const publicClient = getReadOnlyClient();
+    const chainId = Number(chain.chainId);
+    const publicClient = getReadOnlyClient(chainId, BE_ALCHEMY_API_KEY);
```

Comment:
Maybe this should be wrapped in something in api-lib like getBackendReadOnlyClient